sql_assign_3.sql

```diff
@@ -0,0 +1,214 @@
+-- 1. Extract the middle 3 characters from the string 'ABCDEFG'.
+SELECT x = SUBSTRING('ABCDEFG',3,3)
+
+-- 2. From a table 'Employees' with a column 'FullName', write a query to extract the first name (assuming it's always the first word before a space).
+SELECT LEFT(FullName , CHARINDEX(' ',FullName)) as FullName FROM Employees ;
```

Comment:
Here a space will be there at the end of the output right ?, it should be removed, no.

Reply:
> Here a space will be there at the end of the output right ?, it should be removed, no.

Yes, we can enclose LEFT function in RTRIM to remove ending space

Reply:
Is there any other way ?

Reply:
> Is there any other way ?

maybe by changing condition LEFT(FullName ,( CHARINDEX(' ',FullName) -1 ))